### DISCUSSION TITLE
Use potenciometers on internal Pins.

### DISCUSSION THREAD

**Discussion by PeaceDez** [copych/AcidBox#8] [Q&A] [answered]
Hi copych,

Thank you so much for writing this code; it’s a lot of fun to play with. I’ve got the synthesizer sounding and receiving MIDI. I’m sending notes through USB hairless MIDI and trying to change parameters with potentiometers, which are connected to pins 34, 35, 36 as stated in config.h line 37 <const uint8_t POT_PINS[POT_NUM] = {34, 35, 36};>.
However, it’s not working; the pots don’t change anything. Is this feature with pots already implemented in your synth, or am I doing something wrong?

**Comment by copych** [comment, marked as answer]
Yes, you have noticed it right. So check the latest commit that I've just pushed, it features some experimental code. Anyways, It's all a bit experimental )

**Reply by PeaceDez** [reply]
Thank you. Now it's working.
